lib/new-admin/config/accounts.js

```diff
@@ -1,5 +1,6 @@
 const _ = require('lodash/fp')
 const { COINS, ALL_CRYPTOS } = require('./coins')
+const _ = require('lodash/fp')
```

Comment:
This is already required. Why didn't this cause a lint error?

Reply:
Fixed!